packages/graphql-migrations/src/abstract/generateAbstractDatabase.ts

```diff
@@ -234,6 +238,10 @@ class AbstractDatabaseBuilder {
 
       // Object
     } else if (isObjectType(fieldType)) {
+      if (relationshipMarker?.virtual) {
```

Comment:
Why we need virtual relationship marker? It seems overly complicated and reasons are not obvious to me

Reply:
Because in a database a one-to-one relationship is unidirectional always - only one table in a 1:1 will have a foreign key field.

So we need a way to add a 1:1 field to the other model without generating a foreign key field.

This is a virtual one-to-one, as it does not exist in the database.

`@oneToOne key: 'note_id'` ⬅️ this generates a 'note_id' column in the table for this model.

`@oneToOne key: 'note_id', virtual: true` ⬅️ this does not generate a `note_id` column for this model.